.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Is this an accidental checkin? Not everyone uses VSCode, and the settings seem tailored towards debugging here.